alerts/machine-config-operator/OWNERS

```diff
@@ -0,0 +1,14 @@
+approvers:
```

Comment:
https://github.com/openshift/runbooks/blob/master/alerts/machine-config-operator/OWNERS now already exist, so you may need to update existing one. Also, will be good to update it with https://github.com/openshift/machine-config-operator/blob/master/OWNERS